modules/monitoring-configuring-external-alertmanagers.adoc

```diff
@@ -3,144 +3,116 @@
 Currently supported authentication methods are bearer token (`bearerToken`) and client TLS (`tlsConfig`).
-The following sample config map configures an additional Alertmanager using a bearer token with client TLS authentication:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
-data:
-  config.yaml: |
-    prometheusK8s:
-      additionalAlertmanagerConfigs:
-      - scheme: https
-        pathPrefix: /
-        timeout: "30s"
-        apiVersion: v1
-        bearerToken:
-          name: alertmanager-bearer-token
-          key: token
-        tlsConfig:
-          key:
-            name: alertmanager-tls
-            key: tls.key
-          cert:
-            name: alertmanager-tls
-            key: tls.crt
-          ca:
-            name: alertmanager-tls
-            key: tls.ca
-        staticConfigs:
-        - external-alertmanager1-remote.com
-        - external-alertmanager1-remote2.com
-----
-
-** *To configure additional Alertmanager instances for routing alerts from user-defined projects*:
-endif::openshift-dedicated,openshift-rosa[]
-
-.. Edit the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` project:
+// tag::UWM[]
+<2> Substitute `<component>` for one of two supported external Alertmanager components: `prometheus` or `thanosRuler`.
+// end::UWM[]
 +
-[source,terminal]
-----
-$ oc -n openshift-user-workload-monitoring edit configmap user-workload-monitoring-config
-----
-
-.. Add a `<component>/additionalAlertmanagerConfigs:` section under `data/config.yaml/`.
-
-.. Add the configuration details for additional Alertmanagers in this section:
+The following sample config map configures an additional Alertmanager for {component-name} by using a bearer token with client TLS authentication:
 +
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: user-workload-monitoring-config
-  namespace: openshift-user-workload-monitoring
+  name: {configmap-name}
+  namespace: {namespace-name}
 data:
   config.yaml: |
-    <component>:
-      additionalAlertmanagerConfigs:
-      - <alertmanager_specification>
-----
-+
-For `<component>`, substitute one of two supported external Alertmanager components: `prometheus` or `thanosRuler`.
-+
-For `<alertmanager_specification>`, substitute authentication and other configuration details for additional Alertmanager instances. Currently supported authentication methods are bearer token (`bearerToken`) and client TLS (`tlsConfig`). The following sample config map configures an additional Alertmanager using Thanos Ruler with a bearer token and client TLS authentication:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: user-workload-monitoring-config
-  namespace: openshift-user-workload-monitoring
-data:
-  config.yaml: |
-    thanosRuler:
+    {component}:
```

Comment:
My ignorance: Does this need the `tag::CPM[]` and `tag::UWM[]`?

Reply:
I see how this can be confusing. Here it does not need tag, because here we use `{component}` (attribute), while before we used `<component>` and `prometheusK8s` (variables).